### ISSUE TITLE
Whats the acc of this implementation on ImageNet ?

### ISSUE THREAD

**Issue by go2sea** [rishikksh20/convolution-vision-transformers#3]
Thanks for releasing your CVT implementation. 

Can you recurrent the accuracy in the paper?

**Comment by rishikksh20** [comment]
@go2sea Use this repo: https://github.com/rosinality/vision-transformers-pytorch pre-processing and training script for CvT model.
I will try when I have vacant gpus and some extra time.
